admin-dev/filemanager/include/php_image_magician.php

```diff
@@ -2644,13 +2635,6 @@ public function displayImage($fileType = 'jpg', $imageQuality="100")
     # Notes:
     #
   {
-      if (!is_resource($this->imageResized)) {
```

Comment:
If I remember what you said, the problem occurred only with php 8.1.
Then what is the change that was done with php 8.1 that you fix removing this code ?

And then why didn't you remove this line for example  : https://github.com/PrestaShop/PrestaShop/blob/94c5f31ad0f1926b58c0fa02f8a4b7ce5db4c688/admin-dev/filemanager/include/php_image_magician.php#L3148 ?
Did you check if this function was called somewhere else ?

Reply:
Yes, in case you want to resize `.bmp` files, I forgot to remove it.